content/de/includes/user-guide-migration-notice.md

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
It is better to not have English content present as it may change while you're translating.